thunder/tests/distributed/test_fsdp.py

```diff
@@ -841,19 +851,20 @@ def _test_fsdp_transformer_engine_bucketing(input_data):
         torch.cuda.set_device(rank)
 
         # data
-        batch_size = 2
-        max_seq_len = 32
-        vocab_size = 32
```

Comment:
Before this TE [PR](https://github.com/NVIDIA/TransformerEngine/pull/1437), this was the minimum requirement for shape.

Reply:
Right, we should update this based on the recipe. Thanks!

Reply:
Reference for MXFP8 checks - https://github.com/NVIDIA/TransformerEngine/blob/884248f5788ad1d5561a55878cd772866efc46a8/transformer_engine/pytorch/csrc/extensions/quantizer.cpp#L177-L180